### ISSUE TITLE
Reporting

### ISSUE THREAD

**Issue by fniessink** [ICTU/quality-time#5996]
# Epic

As a quality manager, I want the quality report dashboard to provide an overview over a specific period.

## Acceptance criteria

- [ ] Stacked area charts, in addition to bar charts and donut charts. 
- [ ] Add comments to cards in the dashboard (cards will be wider to make room for the text). Card comments can be edited under the subject headers.
- [ ] Cards should be hidable.

## Open issues

- [ ] **Blocking**: Dashboard layout is stored in the database and can't be passed as query parameter: problem for maintaining different layouts.
- [ ] Should the area charts be wider than the bar charts?
- [ ] How to make periods visible in the area charts? Vertical lines to denote the dates?
- [ ] We should discuss in a refinement if it is feasible to remove Subjects completely from the data model. What will be the benefits and what will be the impact. This spike can also be separated from this epic if required.

## Out of scope

- [ ] When the bar charts in the dashboard have two columns, put an icon in between that shows the change (⬂⬀⬄). Open issue: what constitutes worse or better?
- [ ] Make cards resizeable

# User stories

- [x] #7039
- [x] #4519
- [ ] #3582
- [ ] #5345
- [x] #7086
- [ ] #11567 

# Dependencies

The higher story is blocked by the lower stories:

- [x] #7214
- [x] #7215
- [x] #7211
- [x] #7213
- [x] #7212
